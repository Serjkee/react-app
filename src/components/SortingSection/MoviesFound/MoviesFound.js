import React from 'react'

import './MoviesFound.scss'

const MoviesFound = (props) => {
  return (
    <p className='movies-found'>
      {props.moviesFound} movies found
    </p>
  )
}

export default MoviesFound
