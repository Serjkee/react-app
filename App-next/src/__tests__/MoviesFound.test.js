import React from 'react'
import MoviesFound from '../components/SortingSection/MoviesFound/MoviesFound'

describe('Tests MoviesFound component', () => {
  it('Matching snapshot', () => {
    const component = shallow(<MoviesFound />)

    expect(component).toMatchSnapshot();
  })
})
