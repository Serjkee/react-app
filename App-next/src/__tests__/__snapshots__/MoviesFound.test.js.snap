// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests MoviesFound component Matching snapshot 1`] = `
<p
  className="movies-found"
>
   movies found
</p>
`;
